inference/server/oasst_inference_server/routes/configs.py

```diff
@@ -73,3 +87,63 @@ async def get_model_configs() -> list[ModelConfigInfo]:
         for model_config_name in model_configs.MODEL_CONFIGS
         if (settings.allowed_model_config_names == "*" or model_config_name in settings.allowed_model_config_names_list)
     ]
+
+
+@router.post("/plugin_config")
+async def get_plugin_config(plugin: inference.PluginEntry) -> inference.PluginEntry | fastapi.HTTPException:
```

Comment:
Elsewhere in the project we don't type hint exceptions as return types so would be good to change this (and other instances of the same) for consistency